src/components/UserBubble/UserBubble.vue

```diff
@@ -152,33 +240,37 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.user-bubble-popover {
-	display: inline;
-	vertical-align: top;
-}
-
 .user-bubble {
-	display: flex;
-	height: 20px;
-	border-radius: 10px;
-	background-color: var(--color-background-dark);
-}
+	&__wrapper {
+		// align inline with text
+		display: inline-block;
+		vertical-align: middle;
+		// shrink and allow grow to fit
+		min-width: 0;
+		max-width: 100%;
+	}
 
-.user-bubble-primary {
-	color: var(--color-primary-text);
-	background-color: var(--color-primary-element);
-}
+	&__content {
+		display: inline-flex;
+		max-width: 100%;
+		background-color: var(--color-background-dark);
```

Comment:
Now that we have the title slot we should probably have `padding-right` on the content instead of margin-right on the title, as otherwise there is some issue at the right side of the bubble if there is something in the slot:
![image](https://user-images.githubusercontent.com/22591354/85629439-2ca8ac00-b672-11ea-9485-911077571b43.png)

Reply:
Good idea!!